pkg/grizzly/server.go

```diff
@@ -102,13 +102,14 @@ var mustProxyPOST = []string{
 	"/api/ds/query",
 }
 var blockJSONget = map[string]string{
-	"/api/ma/events":    "[]",
-	"/api/live/publish": "[]",
-	"/api/live/list":    "[]",
-	"/api/user/orgs":    "[]",
-	"/api/annotations":  "[]",
-	"/api/search":       "[]",
-	"/api/usage/*":      "[]",
+	"/api/ma/events":       "[]",
+	"/api/live/publish":    "[]",
+	"/api/live/list":       "[]",
+	"/api/user/orgs":       "[]",
+	"/api/annotations":     "[]",
+	"/api/search":          "[]",
+	"/api/usage/*":         "[]",
+	"/api/frontend/assets": "{}",
```

Comment:
I assume this is unrelated and just came up as an endpoint that the proxy needs to capture?

Reply:
yes